dotCMS/src/main/java/com/dotcms/rest/api/v1/accessibility/ACheckerResource.java

```diff
@@ -0,0 +1,237 @@
+
+import com.dotcms.enterprise.achecker.ACheckerResponse;
+import com.dotcms.enterprise.achecker.model.GuideLineBean;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.google.common.annotations.VisibleForTesting;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.glassfish.jersey.server.JSONP;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+import static com.dotcms.rest.api.v1.accessibility.ACheckerHelper.CONTENT;
+import static com.dotcms.rest.api.v1.accessibility.ACheckerHelper.GUIDELINES;
+import static com.dotcms.util.DotPreconditions.checkNotEmpty;
+
+/**
+ * This REST Endpoint provides the ability to validate content against the Accessibility Checker
+ * service.
+ *
+ * @author Jose Castro
+ * @since Sep 3rd, 2024
+ */
+@Path("/v1/achecker")
+@Tag(name = "Accessibility Checker",
+        description = "Endpoints that perform operations related to validating accessibility in content.")
+public class ACheckerResource {
+
+    private final WebResource webResource;
+
+    @SuppressWarnings("unused")
+    public ACheckerResource() {
+        this(new WebResource());
+    }
+
+    @VisibleForTesting
+    public ACheckerResource(final WebResource webResource) {
+        this.webResource = webResource;
+    }
+
+    /**
+     * Returns the list of Accessibility Guidelines available in the system.
+     *
+     * @return The list of Accessibility Guidelines available in the system.
+     *
+     * @throws Exception If the guidelines can't be retrieved.
+     */
+    @GET
+    @Path("/guidelines")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    @Operation(
+            operationId = "getAccessibilityGuidelines",
+            summary = "Retrieves Accessibility Guidelines",
+            description = "Returns the list of available Accessibility Guidelines that are used to validate content.",
+            tags = {"Accessibility Checker"},
+            responses = {
+                    @ApiResponse(responseCode = "200", description = "Guideline list retrieved successfully",
+                            content = @Content(mediaType = "application/json")),
+                    @ApiResponse(responseCode = "500", description = "Internal Server Error")
+            }
+    )
+    public final Response getGuidelines(@Context final HttpServletRequest request,
```

Comment:
do not return Response, return ResponseGuideLinesEntityView extends ResponseEntityView<List<GuideLineBean> >

Reply:
Done!